### ISSUE TITLE
Terminated email from niantic

### ISSUE THREAD

**Issue by IamDeric** [finndev/PokeBuddy#1218]
![image](https://cloud.githubusercontent.com/assets/20781731/17759581/0bad68f8-64ad-11e6-9cb8-3a90a9f27974.png)

**Comment by JoeyIsLit** [comment]
Yea I got my first email too...

**Comment by weilinzung** [comment]
how often you use the bot? And wondering what is in that form.

**Comment by IamDeric** [comment]
The form is to re appeal again. Just to reason out again your mistake. I think i over doing the bot. But i make a new one again. And im around lvl 23 i think. My main is 35 that been terminated. Making alot of new account now wait for trading and then trade everything to a new account without bot at all.

**Comment by IamDeric** [comment]
I just received my 2nd appeal email from them. No chance for getting my main account. Same email as this one.

**Comment by darkuru** [comment]
When did you put in the appeal? Can use that as a little bit of reference when i can expect mine.

**Comment by Rab89** [comment]
You got banned for use a bot that interferes with the pokemongo ToS, i just dont see the need to constantly see these threads started. BooHoo
after all these **# ISSUES** dont have anything to do with the bot,

**Comment by Killer1234123** [comment]
I got this email too, but still can play. Is this normal?

**Comment by darkuru** [comment]
killer1234123,

If that really is the case, that you got the email but still can login and actually just play like normally. 
Avoid doing anything with that email cuz that email sounds fishy as fuck.

**Comment by Killer1234123** [comment]
So i ignore the mail?

**Comment by darkuru** [comment]
you can play normally, that email doesn't concern you.

**Comment by Synrise** [comment]
No clue if it's posted before or not... I hate to ask questions that have been answered already.

I've been banned on 4 account, my main (lvl 29), 2 alts, all on which I botted and a new account that I didn't bot on. So I'm wondering if they ban our account or if it's an actual IP ban, which would mean I'd never be able to play again? :s

**Comment by Kokainium** [comment]
Can i ask one Thing :D

**Comment by Demolition1986** [comment]
@IamDeric xaoxoa but the important pokemon are in the main banned or not? xoaoxao, well i doing like u said, if i get ban, i create one and another and another account again, ill not stop bot.. Just getting 1 lv per day is ok i guess.

**Comment by IamDeric** [comment]
@Demolition1986 , yes it is. But its ok.  @Rab89 did i say its an issue? Read the title if you can read. Im just sharing what niactic send me.

**Comment by Rab89** [comment]
dont get salty ofcourse its not an issue (for me), your posting something 100's of people have over the past few days :laughing:
